src/components/partials/user-editor/user-editor.component.js

```diff
@@ -343,88 +350,45 @@ class UserEditor extends Component {
-                        {authService.userRole === "global" ? (
-                          <MenuItem value="global">
-                            <em>{t("ADMINS.GLOBAL")}</em>
-                          </MenuItem>
-                        ) : (
-                          ""
-                        )}
-                        {authService.userRole === "global" ||
-                        authService.userRole === "agency" ? (
-                          <MenuItem value="agency">
-                            <em>{t("ADMINS.AGENCY")}</em>
-                          </MenuItem>
-                        ) : (
-                          ""
-                        )}
-                        <MenuItem value="group">
-                          <em>{t("ADMINS.GROUP")}</em>
-                        </MenuItem>
-                        <MenuItem value="field">
-                          <em>{t("ADMINS.FIELD")}</em>
-                        </MenuItem>
-                      </Select>
-                    </FormControl>
-                    <div className="error-messages">{t(errors.adminType)}</div>
-                    <FormControl className="form-input">
-                      <InputLabel id="agency-label">
-                        {t("TABLE.AGENCY")}
-                      </InputLabel>
-                      <Select
-                        readOnly={!!showingOptions.readOnly}
-                        labelId="agency-label"
-                        onChange={(e) =>
-                          setFieldValue("agency", e.target.value)
-                        }
-                        value={values.agency}
-                      >
-                        {authService.userRole === "global" ? (
-                          agencies.map((agency, ind) => (
-                            <MenuItem value={agency} key={ind}>
-                              <em>{agency}</em>
-                            </MenuItem>
-                          ))
-                        ) : (
-                          <MenuItem
-                            value={authService.user.agency.name}
-                            key={0}
-                          >
-                            <em>{authService.user.agency.name}</em>
-                          </MenuItem>
-                        )}
-                      </Select>
-                    </FormControl>
-                    <FormControl className="form-input">
-                      <InputLabel id="group-label">
-                        {t("CREATE_USER_PAGE.USER_GROUP")}
-                      </InputLabel>
-                      <Select
-                        readOnly={!!showingOptions.readOnly}
-                        labelId="group-label"
-                        onChange={(e) =>
-                          setFieldValue("userGroup", e.target.value)
-                        }
-                        value={values.userGroup}
-                      >
-                        <MenuItem value="User Group">
```

Comment:
I noticed that the value for this field is not like the data field in roles
lowercase one-word more like a data value
```
 <MenuItem value="global">
<MenuItem value="agency">
<MenuItem value="group">
<MenuItem value="field">
```

this one is `<MenuItem value="User Group">`

I guess usually select data values are identifiers wired to a data store/database, rather than a "name".
Just wondering if this was intended, and if not, might be causing bugs.

I tried to git blame, but it was there on the very first commit
https://github.com/WildAid/o-fish-web/blame/cc84f5986f6b2aafec44a8b8e9f910e4f7bfe23e/src/components/partials/user-editor/user-editor.component.js#L328

Reply:
This is also the reason why the Group value always shows "User Group" instead of the designated i18n translation in the MenuItem `t("CREATE_USER_PAGE.USER_GROUP")` (which is "Group")